tests/test_2d_operation_construction.cpp

```diff
@@ -62,8 +63,9 @@ DEBUG_TriMesh test_collapse(const DEBUG_TriMesh& mesh, const Tuple& e, bool shou
     REQUIRE(m.is_connectivity_valid());
     REQUIRE(should_succeed == result);
     if (should_succeed) {
-        TriMesh m2 = mesh;
-        m2.collapse_edge(e);
+        DEBUG_TriMesh m2 = mesh;
+        Accessor<long> hash_accessor = m2.get_cell_hash_accessor();
```

Comment:
these cases make me think taht collapse_edge_slow and split_edge_slow might be helpful just to remove a bit of this boilerplate